api/src/test/java/uk/gov/dhsc/htbhf/claimant/service/PregnancyEntitlementCalculatorTest.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.dhsc.htbhf.claimant.service;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDate;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class PregnancyEntitlementCalculatorTest {
+
+    private static final int PREGNANCY_GRACE_PERIOD_IN_DAYS = 14;
+
+    PregnancyEntitlementCalculator calculator;
+
+
+    @BeforeEach
+    void setup() {
+        calculator = new PregnancyEntitlementCalculator(PREGNANCY_GRACE_PERIOD_IN_DAYS);
```

Comment:
This line could be moved to where we declare the calculator. Less code and it means we don't create a new object with every test.